src/components/select/select.scss

```diff
@@ -58,13 +58,14 @@
 
         &:focus {
             outline: none;
-            .mdc-select__dropdown-icon {
-                box-shadow: var(--button-shadow-hovered);
-            }
+
             .mdc-floating-label {
                 color: var(--mdc-theme-primary);
             }
         }
+        &:focus-visible {
+            box-shadow: var(--shadow-depth-8-focused) !important;
```

Comment:
I'm guessing the following isn't the intended look?

https://user-images.githubusercontent.com/154725/110787915-3872e780-826e-11eb-8269-81defc37a501.mov

Reply:
(When there's no drop-shadow is when the element is focused using the keyboard.)

Reply:
Ah‚ I now see that it requires #1179. I had missed that.